src/features/home/l10n/messageIds.ts

```diff
@@ -45,6 +45,13 @@ export default makeMessages('feat.home', {
   footer: {
     privacyPolicy: m('Privacy policy'),
   },
+  settings: {
+    lang: {
+      auto: m('Automatic (from browser)'),
```

Comment:
maybe "same as browser" instead of "from browser" is a bit more clear, but this is details.